### PR TITLE
Add simple KPI metric visualization notebook

### PR DESCRIPTION
**Pull request by hemajv**
## Related Issues and Dependencies
#217
#275 

## This introduces a breaking change

- [ ] Yes
- [X] No

## Description

Adding a simple KPI metric visualization notebook that plots a time series graph for each metric aggregated over all TestGrid dashboards, jobs and tests i.e. aggregated to the entire platform level

### CODE REVIEW

File: notebooks/data-sources/TestGrid/metrics/metric_visualization_generic.ipynb [hunk 1]
```diff
@@ -0,0 +1,1309 @@
+{
```
Comment (isabelizimm) [inline notebooks/data-sources/TestGrid/metrics/metric_visualization_generic.ipynb:1]: <p>small nit: can the graphs with overlapping date labels be rotated for readability?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

**Review by aakankshaduggal — COMMENTED** [review]
Overall looks good to me. Just a small suggestion, if you could fix the ticks and add a conclusion to the notebook it will be great.

**Review by chauhankaranraj — COMMENTED** [review]
/lgtm :+1:

File: notebooks/data-sources/TestGrid/metrics/metric_visualization_generic.ipynb (hunk 1 above)
Comment (hemajv) [reply notebooks/data-sources/TestGrid/metrics/metric_visualization_generic.ipynb:1]: updated :+1:

**Comment by hemajv** [comment]
> Overall looks good to me. Just a small suggestion, if you could fix the ticks and add a conclusion/future steps to the notebook it will be great.

thanks for pointing that out! updated the graph ticks and added a conclusion for the notebook as well as the graphs.

**Comment by MichaelClifford** [comment]
:rocket: :rocket: :rocket:  Thanks @hemajv this is great!

**Pull request merged**